lib/rubocop/cop/rails/file_path.rb

```diff
@@ -98,6 +98,7 @@ def check_for_extension_after_rails_root_join_in_dstr(node)
         def check_for_file_join_with_rails_root(node)
           return unless file_join_nodes?(node)
           return unless node.arguments.any? { |e| rails_root_nodes?(e) }
+          return if node.arguments.any?(&:variable?)
```

Comment:
String literals starting with `/` should also be ignored when the argument is a string literal. Can you update it and add tests?

Reply:
Cases where the string contains two or more `//` should also be ignored.

```ruby
rails8-app(dev)> default_url = "default_images/ofn-logo.png"
rails8-app(dev)> Rails.root.join('public//', default_url).to_s
=> "/tmp/example/public/default_images/ofn-logo.png"
rails8-app(dev)> File.join(Rails.root, 'public//', default_url)
=> "/tmp/example/public//default_images/ofn-logo.png"
```

Reply:
I've done it.